insights/parsers/mysqladmin.py

```diff
@@ -15,6 +18,42 @@
 from insights.specs import Specs
 
 
+@parser(Specs.mysqladmin_status)
+class MysqladminStatus(LegacyItemAccess, CommandParser):
+    """
+    Module for parsing the output of the ``mysqladmin status`` command.
+
+    Typical output looks like:
```

Comment:
@asyedham there should be a double colon at the end of this line (`...looks like::`)to properly format the following text.

Reply:
@**bfahr**  Done